libyul/AsmAnalysis.cpp

```diff
@@ -705,6 +706,9 @@ bool AsmAnalyzer::validateInstructions(evmasm::Instruction _instr, SourceLocatio
 		);
 	};
 
+
+	// The errors below are only issued when the undeclared identifier is a builtin on the default EVM version
+	// but not on the currently selected one, since `validateInstructions` uses the default EVM version.
```

Comment:
Nice. I'd just clarify it a bit since there are two functions called `validateInstructions()` and this one does not actually use the default EVM version.

```suggestion
	// The errors below are meant to be issued when processing an undeclared identifier matching a builtin name present on the default EVM version
	// but not on the currently selected one, since the other `validateInstructions()` overload uses the default EVM version.
```